packages/scene-composer/src/utils/entityModelUtils/SceneLifecycleModule.ts

```diff
@@ -0,0 +1,32 @@
+import { GetSceneCommandOutput } from '@aws-sdk/client-iottwinmaker';
+import { SceneMetadataModule } from '@iot-app-kit/source-iottwinmaker';
+
+import { LAYER_ROOT_ENTITY_ID, SCENE_ROOT_ENTITY_COMPONENT_NAME, LayerType } from '../../common/entityModelConstants';
+import { SceneCapabilities } from '../../common/sceneModelConstants';
+import { KnownSceneProperty } from '../../interfaces';
+import { emptyScene } from '../emptyScene';
+import { createLayer } from '../../utils/entityModelUtils/sceneLayerUtils';
+
+import { updateSceneEntityComponent } from './sceneComponent';
+
+export class SceneLifecycleModule {
+  createDynamicScene = async (sceneMetadataModule: SceneMetadataModule): Promise<GetSceneCommandOutput> => {
+    if (!sceneMetadataModule) {
+      throw Error('SceneMetadataModule is undefined');
+    }
```

Comment:
this will be unnecessary since it's a parameter now

Reply:
Removed this!